client/marketing/components/slider/index.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import { Component, createRef } from '@wordpress/element';
+import classnames from 'classnames';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss'
+
+class Slider extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			height: null,
+		};
+		this.container = createRef();
+		this.onExited = this.onExited.bind( this );
+		this.onEnter = this.onEnter.bind( this );
+		this.updateSliderHeight = this.updateSliderHeight.bind( this );
+	}
+
+	/**
+	 * Update the slider height on Resize
+	 */
+	componentDidMount() {
+		// Update the slider height on Resize
+		window.addEventListener( 'resize', this.updateSliderHeight );
+	}
+
+	componentWillUnmount() {
+		window.removeEventListener( 'resize', this.updateSliderHeight )
+	}
+
+	updateSliderHeight() {
+		const slide = this.container.current.getElementsByClassName( 'woocommerce-marketing-slider__slide' )[ 0 ];
```

Comment:
or simply `querySelector( '.woocommerce-marketing-slider__slide' )` that will get you the first result.

Reply:
Thanks for the tip!